elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -572,6 +572,26 @@ public void bindTrigger(Class<? extends Annotation> annotationClass,
         return annotation == NO_ANNOTATION ? null : annotationClass.cast(annotation);
     }
 
+    /**
+     * Return annotation for provided method.
+     *
+     * @param annotationClass the annotation class
+     * @param method the method
+     * @param <A> annotation type
+     * @return the annotation
+     */
+    public <A extends Annotation> A getAnnotation(Class<A> annotationClass, String method) {
```

Comment:
We have a lot of getAnnotation methods.  This should be renamed to getMethodAnnotation.

Reply:
Sure